### ISSUE TITLE
native build

### ISSUE THREAD

**Issue by ggrebert** [quarkiverse/quarkus-unleash#12]
The extension doesn't works in native mode.
Quarkus can't start:

```
__  ____  __  _____   ___  __ ____  ______ 
 --/ __ \/ / / / _ | / _ \/ //_/ / / / __/ 
 -/ /_/ / /_/ / __ |/ , _/ ,< / /_/ /\ \   
--\___\_\____/_/ |_/_/|_/_/|_|\____/___/   
2021-02-24 19:36:39,215 INFO  [no.fin.unl.rep.ToggleBackupHandlerFile] (main) Unleash will try to load feature toggle states from temporary backup
2021-02-24 19:36:39,216 ERROR [io.qua.run.Application] (main) Failed to start application (with profile prod): java.lang.NullPointerException
        at java.util.concurrent.ConcurrentHashMap.putVal(ConcurrentHashMap.java:1011)
        at java.util.concurrent.ConcurrentHashMap.put(ConcurrentHashMap.java:1006)
        at no.finn.unleash.repository.ToggleCollection.<init>(ToggleCollection.java:19)
        at no.finn.unleash.repository.JsonToggleCollectionDeserializer.deserializeVersion1(JsonToggleCollectionDeserializer.java:67)
        at no.finn.unleash.repository.JsonToggleCollectionDeserializer.deserialize(JsonToggleCollectionDeserializer.java:31)
        at no.finn.unleash.repository.JsonToggleCollectionDeserializer.deserialize(JsonToggleCollectionDeserializer.java:14)
        at com.google.gson.internal.bind.TreeTypeAdapter.read(TreeTypeAdapter.java:69)
        at com.google.gson.Gson.fromJson(Gson.java:932)
        at com.google.gson.Gson.fromJson(Gson.java:870)
        at no.finn.unleash.repository.JsonToggleParser.fromJson(JsonToggleParser.java:23)
        at no.finn.unleash.repository.ToggleBackupHandlerFile.read(ToggleBackupHandlerFile.java:33)
        at no.finn.unleash.repository.FeatureToggleRepository.<init>(FeatureToggleRepository.java:46)
        at no.finn.unleash.repository.FeatureToggleRepository.<init>(FeatureToggleRepository.java:26)
        at no.finn.unleash.DefaultUnleash.defaultToggleRepository(DefaultUnleash.java:45)
        at no.finn.unleash.DefaultUnleash.<init>(DefaultUnleash.java:53)
        at io.quarkiverse.unleash.runtime.UnleashCreator.createUnleash(UnleashCreator.java:37)
        at io.quarkiverse.unleash.runtime.UnleashProducer.initialize(UnleashProducer.java:16)
        at io.quarkiverse.unleash.runtime.UnleashProducer_ClientProxy.initialize(UnleashProducer_ClientProxy.zig:215)
        at io.quarkiverse.unleash.runtime.UnleashRecorder.initializeProducers(UnleashRecorder.java:11)
        at io.quarkus.deployment.steps.UnleashProcessor$configureRuntimeProperties186348976.deploy_0(UnleashProcessor$configureRuntimeProperties186348976.zig:87)
        at io.quarkus.deployment.steps.UnleashProcessor$configureRuntimeProperties186348976.deploy(UnleashProcessor$configureRuntimeProperties186348976.zig:40)
        at io.quarkus.runner.ApplicationImpl.doStart(ApplicationImpl.zig:620)
        at io.quarkus.runtime.Application.start(Application.java:90)
        at io.quarkus.runtime.ApplicationLifecycleManager.run(ApplicationLifecycleManager.java:100)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:66)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:42)
        at io.quarkus.runtime.Quarkus.run(Quarkus.java:119)
        at io.quarkus.runner.GeneratedMain.main(GeneratedMain.zig:29)
```

**Comment by ggrebert** [comment]
tested and failed with latest tag (`0.1.0`) and latest commit 358c19d

**Comment by andrejpetras** [comment]
@ggrebert support for native images is fixed by  #40

**Comment by andrejpetras** [comment]
Release `0.2.0` 9d919db
